pkg/config/config.go

```diff
@@ -68,10 +68,16 @@ type TencentCloudConfig struct {
 	SecretID string
 	// Secret Key of Tencent Cloud
 	SecretKey string
+	// VpcID used in TencentCloud CFS storage
+	VpcID string
+	// SubnetID used in TencentCloud CFS storage
+	SubnetID string
 }
 
 // AddFlags add the TencentCloud configurations to global flag.
 func (c *TencentCloudConfig) AddFlags() {
 	flag.StringVar(&c.SecretID, "tencent-cloud-secret-id", "admin", "API Secret ID of Tencent Cloud")
 	flag.StringVar(&c.SecretKey, "tencent-cloud-secret-key", "admin", "API Secret Key of Tencent Cloud")
+	flag.StringVar(&c.VpcID, "tencent-cloud-cfs-vpc-id", "vpc-xxxxxxxx", "VpcID used in TencentCloud CFS storage")
```

Comment:
We also need to config these options for csi-operator in [tke](https://github.com/tkestack/tke).